packages/ember-glimmer/lib/utils/bindings.js

```diff
@@ -14,6 +14,17 @@ function referenceForKey(component, key) {
 }
 
 function referenceForParts(component, parts) {
+  let isAttrs = parts[0] === 'attrs';
+
+  // TODO deprecate this
+  if (isAttrs) {
+    parts.shift();
+
+    if (parts.length === 1) {
+      return referenceForKey(component, parts[0]);
+    }
```

Comment:
We want this to fall through. So given `attrs.foo` we want to delegate to `referenceForKey`, however if we shift off the value and we still have a path e.g. `parts.length !== 1` (guranteed to never be 0) then we want to still create a path ref.